### PR TITLE
Move installation docs to top-level guide

### PR DESCRIPTION
**Pull request by GeoSander**
One of the first things a user needs to know is how to install the software. The current installation documentation (which is incomplete!) can be found in the _Maintainer guide_, but in consultation with @ticheler, we have decided that it is better when there's a top-level _Installation guide_ for this. This way, a user can quickly find the installation docs.

Following the upcoming changes to the installer (see [this GeoNetwork PR](https://github.com/geonetwork/core-geonetwork/pull/4961)), I have added docs for the [new Windows setup](https://my.geocat.net/download/category/6/GeoNetwork.html) and the ZIP distribution. 
Furthermore, I have updated the doc page on the WAR file and fixed a number of typos all across the docs.

### CODE REVIEW

**Comment by GeoSander** [comment]
I also noticed that the path to Modernizr.min.js is incorrect for all HTML pages inside a subfolder (e.g. _install-guide_).
The path was hard-coded in the layout.html, but I now replaced it for the _topath()_ function.

File: source/install-guide/installing-from-zip.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.. _installing-from-zip:
+
+Installing using a ZIP file (Linux/macOS)
```
Comment (fxprunayre) [inline source/install-guide/installing-from-zip.rst:3]: You can remove `Linux/macOS`, this works for everyone.

File: source/install-guide/installing-from-the-installer.rst [hunk 2]
```diff
@@ -0,0 +1,162 @@
+.. _installing-from-the-installer:
```
Comment (fxprunayre) [inline source/install-guide/installing-from-the-installer.rst:1]: The community is not in capacity of building the installer (see https://github.com/geonetwork/core-geonetwork/pull/4961) , so it makes more sense to move that doc to the place where people get this installer https://my.geocat.net ?

File: source/install-guide/index.rst [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
-Before using |project_name| on your own computer, you need to install it.
-There is different ways to install |project_name|:
+Before you can use GeoNetwork on your own computer, you need to install it.
+There are several different ways to install GeoNetwork:
 
 
 .. toctree::
    :maxdepth: 2
 
    installing-from-the-installer.rst
```
Comment (fxprunayre) [inline source/install-guide/index.rst:18]: I would remove that. The website download page https://geonetwork-opensource.org/downloads.html can have link to some other packages

File: source/administrator-guide/configuring-the-catalog/user-interface-configuration.rst [hunk 4]
```diff
@@ -36,7 +36,7 @@ General options
 Footer
 ------
 
-- **Footer**: Select this checkbox to determine whether the |project_name| footer is shown. If not set, no footer will be visible.
```
Comment (fxprunayre) [inline source/administrator-guide/configuring-the-catalog/user-interface-configuration.rst:39]: This variable was here for some reasons (but is probably not used anymore ? - at least not by the project using it some years ago) so no objection to remove.

**Review by fxprunayre — CHANGES_REQUESTED** [review]
Thanks for the cleanup @GeoSander.

File: source/install-guide/installing-from-zip.rst (hunk 1 above)
Comment (GeoSander) [reply source/install-guide/installing-from-zip.rst:3]: Yes, that's true, especially if I remove the Windows installer section

File: source/install-guide/installing-from-the-installer.rst (hunk 2 above)
Comment (GeoSander) [reply source/install-guide/installing-from-the-installer.rst:1]: Yes, I just discussed that with @josegar74 and I will remove this section.

File: source/install-guide/index.rst (hunk 3 above)
Comment (GeoSander) [reply source/install-guide/index.rst:18]: As mentioned above, I will remove it and point to the download section on the GeoNetwork website.
Shortly, I will therefore make a PR for the GeoNetwork website repo as well.

File: source/administrator-guide/configuring-the-catalog/user-interface-configuration.rst (hunk 4 above)
Comment (GeoSander) [reply source/administrator-guide/configuring-the-catalog/user-interface-configuration.rst:39]: The **|project_name|** variable you mean? It is still defined in the _conf.py_ actually, but it doesn't play well in cases where the variable is used inside some kind of markup (bold, italics, hyperlink etc.). For the sake of consistency, I decided to replace it everywhere. It's not that the project name is likely to change soon and even if it is, it'll be just a matter of doing a (careful) find/replace on "GeoNetwork".

File: source/install-guide/index.rst (hunk 3 above)
Comment (GeoSander) [reply source/install-guide/index.rst]: Done: https://github.com/geonetwork/website/pull/28

**Review by fxprunayre — APPROVED** [review]

**Review by ticheler — APPROVED** [review]
Thanks @GeoSander!

**Pull request merged**